hal/ti_hercules.ld

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
This file appears duplicated in `IDE/CCS/TMS570LC43xx/target.ld`? Would be neat if TI CCS could use this one, otherwise two is fine, but might be worth documenting the differences (one with our make cross and one with TI).

Reply:
This version allows for different config settings with substitution of `WOLFBOOT_PATITION_BOOT_ADDRESS`. Not sure how to support the same with the IDE.